api/accounts/accounts_test.go

```diff
@@ -3,11 +3,12 @@
 // Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
 // file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
 
-package accounts_test
+package accounts
```

Comment:
Could you elaborate on the reason behind this? is it possible to keep `accounts_tests`?

Reply:
The same reason [here](https://github.com/vechain/thor/pull/676#discussion_r1542519664)